process/archiving.md

```diff
@@ -2,29 +2,40 @@
 
 Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC, or the TOC may no longer wish to recommend the use of a project.
 
-## Archiving Criteria
+## Triggering Archiving review 
 
-When voting on a proposal to archive a project, TOC members may wish to consider whether the project continues to meet the criteria for CNCF acceptance. The TOC may also look at activity levels in the project (https://all.devstats.cncf.io/d/53/projects-health-table?orgId=1), although it is important to note that there is a difference between a mature project that doesn't get much attention any more but is stable, versus a project that is inactive.
+CNCF TOC may also look at a variety of signals, including but not limited to:
+* [CNCF Project health dashboard](https://projecthealth.cncf.io/)
+* If the project would currently meet criteria for acceptance to CNCF
+* Requests for archival by anyone in the wider CNCF community
+* Security posture of the project, in particular speedy reaction to and resolution of vulnerability reports
+* Project meetings schedules and notes
+* Public mailing list activity
+* Adoption stats
 
-## Voting Process
+Automated systems and the references above can trigger a review by the TCO, for example a project might be very mature and close to feature complete, which would be different from general inactiveness. The archival process itself remains manual with close attention to the details by TOC.
 
-To archive a project:
+## Archiving Process
 
-* A proposal must be put forth to the TOC repo
-* The TOC will inform the project maintainers, CNCF End-User community and wider community of all archiving proposals
-* The proposal must remain open for at least 2 weeks of discussion after the maintainers are informed.
-* A vote must be finalized with 2/3 approval from the TOC
+This is a multi-step process with ample time for projects to react, discuss, and improve built in. Being slow and deliberate is a bug, not a feature. The process can be stopped by TOC consensus at any time until a formal vote has been announced.
 
-## Archiving Process
+1. Archiving review is triggered (see above).
+2. TOC opens an issue inquiring about the health of the project. While there's not fixed timeline, it will be in the order of months.
+3. A formal archival request is created in https://github.com/cncf/toc/issues and the issue tracker of the member project, cross-linking both.
+4. TOC informs project maintainers and the end user community. This step will usually be delegated to CNCF staff.
+5. A mandatory two week discussion phase begins
```

Comment:
nit: s/discussion/public comment period

to stay consistent with nomenclature with other TOC processes

Reply:
Good catch, will update, thanks!